core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -513,7 +513,8 @@ public String toString() {
   public static final PropertyKey UNDERFS_ADDRESS =
       new Builder(Name.UNDERFS_ADDRESS)
           .setDefaultValue(String.format("${%s}/underFSStorage", Name.WORK_DIR))
-          .setDescription("Alluxio directory in the under file system.")
+          .setDescription("Under file storage address. This property is deprecated; "
+              + "use alluxio.master.mount.table.root.ufs instead")
```

Comment:
not sure here. `alluxio.master.mount.table.root.ufs` is too long, maybe have `alluxio.underfs.address` an alias ?